package.json

```
@@ -22,7 +22,7 @@
     "posttest": "eslint .",
     "mocha": "cross-env BABEL_ENV=test NODE_PATH=./src nyc -s mocha -R dot --recursive -t 5s",
     "test": "npm run mocha tests/src",
-    "test-compiled": "npm run prepublish && NODE_PATH=./lib mocha --compilers js:babel-register --recursive tests/src",
+    "test-compiled": "npm run prepublish && cross-env NODE_PATH=./lib mocha --compilers js:babel-register --recursive tests/src",
```

This is a completely unrelated fix. I needed this to run the script in my Windows environment.

My suggestion is to remove it from this pull request.
```suggestion
    "test-compiled": "npm run prepublish && NODE_PATH=./lib mocha --compilers js:babel-register --recursive tests/src",
```